node/src/databroker.rs

```diff
@@ -362,7 +362,7 @@ impl DataBrokerSrv {
             return Err(anyhow::anyhow!("no items to fetch"));
         }
 
-        Ok(Message::new_get_data(inv))
+        Ok(Message::new_inv(inv))
```

Comment:
If so, we should change the Topic in "Inv" instead of "GetData". I just changed to follow the convention used where the new refers to the topic of the message